arm-sql/2014-04-01/swagger/recoverableDatabases.json

```diff
@@ -0,0 +1,351 @@
+        "edition": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The edition of the Azure SQL Database."
+        },
+        "serviceLevelObjective": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current service level objective name for the database."
+        },
+        "elasticPoolName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The elastic pool name for the database."
+        },
+        "lastAvailableBackupDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The last available backup date (ISO8601 format)."
+        }
+      },
+      "description": "Represents an Azure SQL Database geo backup."
+    },
+    "GeoBackupList": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeoBackup"
+          },
+          "description": "Gets or sets the list of a given Azure SQL Database geo backups."
+        }
+      },
+      "description": "Represents the response to a List Azure SQL Database geo backups request."
+    },
+    "DeletedDatabaseBackup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DeletedDatabaseBackupProperties",
+          "description": "Gets or sets the object representing a given Azure SQL Database deleted database backup."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/SqlSubResource"
+        }
+      ],
+      "description": "Represents an Azure SQL Database restorable deleted database."
+    },
+    "DeletedDatabaseBackupProperties": {
+      "properties": {
+        "databaseName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Azure SQL Database name."
+        },
+        "creationDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The creation date of the Azure SQL Database (ISO8601 format)."
+        },
+        "deletionDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The deletion date of the Azure SQL Database (ISO8601 format)."
+        },
+        "earliestRestoreDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The recovery period starte date of the Azure SQL Database. This records the start date and time when recovery is available for this Azure SQL Database (ISO8601 format)."
+        },
+        "edition": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The edition of the Azure SQL Database.",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
```

Comment:
in case of the Database_Create this enum has more values, like Free, Stretch and Datawarehouse